internal/resources/grafana/resource_group_attribute_mapping.go

```diff
@@ -0,0 +1,241 @@
+package grafana
+
+import (
+	"context"
+	"strconv"
+
+	goapi "github.com/grafana/grafana-openapi-client-go/client"
+	"github.com/grafana/grafana-openapi-client-go/models"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"github.com/grafana/terraform-provider-grafana/v3/internal/common"
+)
+
+var (
+	_ resource.ResourceWithImportState = (*resourceGroupAttributeMapping)(nil)
+
+	resourceGroupAttributeMappingName = "grafana_group_attribute_mapping"
+	resourceGroupAttributeMappingID   = common.NewResourceID(common.OptionalIntIDField("orgID"), common.StringIDField("group_id"))
+)
+
+func makeResourceGroupAttributeMapping() *common.Resource {
+	return common.NewResource(
+		common.CategoryGrafanaEnterprise,
+		resourceGroupAttributeMappingName,
+		resourceGroupAttributeMappingID,
+		&resourceGroupAttributeMapping{},
+	).WithLister(listerFunctionOrgResource(listGroupAttributeMappings))
+}
+
+type resourceGroupAttributeMappingModel struct {
+	ID       types.String   `tfsdk:"id"`
+	OrgID    types.String   `tfsdk:"org_id"`
+	GroupID  types.String   `tfsdk:"group_id"`
+	RoleUIDs []types.String `tfsdk:"role_uids"`
```

Comment:
I think that it should be `types.Set` instead. I don't find `[]types.String` any other place.